content/docs/administration/troubleshooting/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Troubleshooting"
+title: "Troubleshoot"

```

Comment:
The title now does not match the directory. Does the directory appear in the URL?

Reply:
It does. I will update the directory name.